src/PCRE2Engine.cpp

```diff
@@ -0,0 +1,46 @@
+#include <rematch/compile.h>
+#include "PCRE2Engine.h"
+
+using namespace regexbench;
+
+PCRE2Engine::PCRE2Engine() {}
+
+PCRE2Engine::~PCRE2Engine() {}
```

Comment:
Use default constructor/destructor (`= default` in the class definition) rather than define empty ones.